### PR TITLE
43566 user_view_after_logging,show_word_list

### PR DESCRIPTION
**Pull request by yasin-buet**
Buttons are dummy.Will work that later

### CODE REVIEW

File: app/Http/Controllers/CategoryViewController.php [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+use App\Http\Requests;
+use App\Http\Controllers\Controller;
+use App\Category;
+
+class CategoryViewController extends Controller
```
Comment (KhanhLD) [inline app/Http/Controllers/CategoryViewController.php]: bad naming

File: app/Http/Controllers/LessonStratController.php [hunk 2]
```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+use App\Http\Requests;
+use App\Http\Controllers\Controller;
+
+class LessonStratController extends Controller
```
Comment (KhanhLD) [inline app/Http/Controllers/LessonStratController.php]: bad naming

File: app/Http/Controllers/UserViewController.php [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+use App\Http\Requests;
+use App\Http\Controllers\Controller;
+use App\User;
+
+class UserViewController extends Controller
```
Comment (KhanhLD) [inline app/Http/Controllers/UserViewController.php]: bad naming

File: app/Http/Controllers/UserViewController.php [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $users=User::all();
```
Comment (KhanhLD) [inline app/Http/Controllers/UserViewController.php]:
```php
$users = User::all();
```

File: app/Http/Controllers/UserViewController.php [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+        return view('home',compact ('users'));
```
Comment (KhanhLD) [inline app/Http/Controllers/UserViewController.php]:
```php
return view('home', compact('users'));
```

File: database/migrations/2015_11_09_112110_create_sessions_table.php [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('sessions', function (Blueprint $table) {
+            $table->string('id')->unique();
```
Comment (KhanhLD) [inline database/migrations/2015_11_09_112110_create_sessions_table.php]: why do you not use
```php
$table->increments('id');
```

File: database/migrations/2015_11_09_112110_create_sessions_table.php [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+            $table->text('payload');
+            $table->integer('last_activity');
```
Comment (KhanhLD) [inline database/migrations/2015_11_09_112110_create_sessions_table.php]: no timestamps?

last_activity as integer?

File: resources/views/home.blade.php [hunk 8]
```diff
@@ -1,5 +1,38 @@
...
+                        <a href="{{ URL::to('lesson') }}">Lessons</a>
+                    </h2>
+                </mark>
+            </strong>
+        </div>
+    </div>
+    <div class="row">
+        <div class="col-md-4">
+            <h3>
+                <?php
```
Comment (KhanhLD) [inline resources/views/home.blade.php]: use blade directive please

File: resources/views/word_list.blade.php [hunk 9]
```diff
@@ -1,5 +1,84 @@
...
+            <div class ="row-fluid">
+                <div class="col-md-2">
+                </div>
+                <div class="col-md-5">
+                    <?php echo $word->english_word?>
+                </div>
+                <div class="col-md-5">
+                    @foreach( $meanings as $meaning )
+                        @if ($word->id==$meaning->id)
+                            <?php echo $meaning->bengali_meaning?>
```
Comment (KhanhLD) [inline resources/views/word_list.blade.php]: use blade directive

File: app/Http/Controllers/CategoryViewController.php (hunk 1 above)
Comment (yasin-buet) [inline app/Http/Controllers/CategoryViewController.php]: should i rename with CategoryController ?

File: database/migrations/2015_11_09_112110_create_sessions_table.php (hunk 6 above)
Comment (yasin-buet) [inline database/migrations/2015_11_09_112110_create_sessions_table.php]: i used it  from laravel documentation's example

File: resources/views/word_list.blade.php (hunk 9 above)
Comment (yasin-buet) [inline resources/views/word_list.blade.php]: done in new push.

File: app/Http/routes.php [hunk 10]
```diff
@@ -20,15 +20,12 @@
     Route::get('register', 'AuthController@getRegister');
     Route::post('register', 'AuthController@postRegister');
 });
+Route::resource('user_view','UserController');
```
Comment (KhanhLD) [inline app/Http/routes.php]: if there is no good reasons, you should make the name of resource the same as Controller name
for example: UsersController => resource users, users view folder

and please make the controller name plural noun of model name, UserModel => UsersController

File: database/migrations/2015_11_09_112110_create_sessions_table.php [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+class CreateSessionsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('sessions', function (Blueprint $table) {
```
Comment (KhanhLD) [inline database/migrations/2015_11_09_112110_create_sessions_table.php]: what will you use this table for?

File: resources/views/home.blade.php [hunk 12]
```diff
@@ -1,5 +1,37 @@
...
+                        <a href="{{ URL::to('lesson') }}">Lessons</a>
+                    </h2>
+                </mark>
+            </strong>
+        </div>
+    </div>
+    <div class="row">
+        <div class="col-md-4">
+            <h3>
+                @if(Auth::check()){
```
Comment (KhanhLD) [inline resources/views/home.blade.php]: convention error

File: database/migrations/2015_11_09_112110_create_sessions_table.php (hunk 11 above)
Comment (yasin-buet) [inline database/migrations/2015_11_09_112110_create_sessions_table.php]: thought about fetching session infos. 
haven't needed that yet.

Comment (KhanhLD) [inline database/migrations/2015_11_09_112110_create_sessions_table.php]: do you want to use database table to store session data?
did you make the necessary adjustment to do it?

File: app/Http/routes.php [hunk 13]
```diff
@@ -20,24 +20,18 @@
     Route::get('register', 'AuthController@getRegister');
     Route::post('register', 'AuthController@postRegister');
 });
+Route::resource('users','UserController');
```
Comment (KhanhLD) [inline app/Http/routes.php]: still no changes in Controllers name? did you read about RESTful?

File: app/Http/Controllers/WordController.php [hunk 14]
```diff
@@ -0,0 +1,91 @@
...
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $words = Question::all();
+        $meanings = Answer::all();
+        return view('words', compact('words','meanings'));
```
Comment (KhanhLD) [inline app/Http/Controllers/WordController.php]: add 1 space after ,
I have commented this before. please read coding convention again.

File: resources/views/words.blade.php [hunk 15]
```diff
@@ -0,0 +1,84 @@
...
+        @foreach($words as $word )
+            <div class="row-fluid">
+                <div class="col-md-2">
+                </div>
+                <div class="col-md-5">
+                    {{ $word->english_word }}
+                </div>
+                <div class="col-md-5">
+                    @foreach($meanings as $meaning )
+                        @if($word->id==$meaning->id)
```
Comment (KhanhLD) [inline resources/views/words.blade.php]: still error.

File: app/Http/routes.php (hunk 13 above)
Comment (yasin-buet) [inline app/Http/routes.php]: i actually used it reading from laravel documentation example.

File: database/migrations/2015_11_09_112110_create_sessions_table.php (hunk 11 above)
Comment (yasin-buet) [inline database/migrations/2015_11_09_112110_create_sessions_table.php]: yes i want to use database to store session data.
I made adjustments as said in laravel documentation like configuration and other driver pre-requisite i.e. databse config to use session data.

File: resources/views/words.blade.php [hunk 16]
```diff
@@ -0,0 +1,84 @@
...
+</div>
+    @foreach($words as $word )
+        <div class="row-fluid">
+            <div class="col-md-2">
+            </div>
+            <div class="col-md-5">
+                {{ $word->english_word }}
+            </div>
+            <div class="col-md-5">
+                @foreach($meanings as $meaning )
```
Comment (KhanhLD) [inline resources/views/words.blade.php]:
```php
@foreach ($meanings as $meaning)
    @if ($word->id == $meaning->id)
        {!! $meaning->bengali_meaning !!}
    @endif
@endforeach
```

Comment (KhanhLD) [inline resources/views/words.blade.php]: and please read about displaying data in blade http://laravel.com/docs/5.1/blade#displaying-data

File: resources/views/home.blade.php [hunk 17]
```diff
@@ -1,5 +1,36 @@
...
+                    <a href="{{ URL::to('lessons') }}">Lessons</a>
+                </h2>
+            </mark>
+        </strong>
+    </div>
+</div>
+<div class="row">
+    <div class="col-md-4">
+        <h3>
+            @if(Auth::check())
```
Comment (KhanhLD) [inline resources/views/home.blade.php]:
```php
@if (Auth::check())
    {{ Auth::user()->name }}
@endif
```

File: resources/views/words.blade.php (hunk 16 above)
Comment (yasin-buet) [inline resources/views/words.blade.php]: ok i am on it

**Comment by dinhhoanglong91** [comment]
ok

**Pull request merged**